Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -124,7 +124,7 @@ public final class PIFBuilder {
 
             let workspace = PIF.Workspace(
                 guid: "Workspace:\(rootPackage.path.pathString)",
-                name: rootPackage.manifestName,  // TODO: use identity instead?
+                name: rootPackage.manifest.displayName,  // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
It probably affects path components of composed temporary paths, etc, so we'd at least need to mangle away any path separator.

Reply:
This isn't really user-visible, but I would still think that the display name is what we want to pass here.